azure-kusto-data/setup.py

```diff
@@ -34,9 +34,10 @@
         "Development Status :: 4 - Beta",
         "Intended Audience :: Developers",
         "Topic :: Software Development",
-        "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Programming Language :: Python :: 3.10",
```

Comment:
I'm not familiar with setup.py, but why are we explicitly listing versions beyond the minimum? Does this prevent using version 3.11 once it comes out until these 2 setup.pys are manually upgraded?

Reply:
These are just tags that you can search with in pypi.
So if the user for example searches for packages with the "3.6" classifier they will not find us.

It does mean we need to keep it updated.

Reply:
Is there no way to dynamically indicate we support v3.6+ without having to manually update?

Reply:
Not that I found (and we support 3.7+)